### PR TITLE
Wfp 2564 spelling mistake

### PR DESCRIPTION
**Pull request by sambailey-moj**

### CODE REVIEW

File: app/views/includes/omic-organisation-overview.njk [hunk 1]
```diff
@@ -4,7 +4,7 @@
                 <th class="govuk-table__header sln-table-org-level" aria-sort="none" id="name">{{ childOrganisationLevelDisplayText }}</th>
                 <th class="govuk-table__header govuk-table__header--numeric" aria-sort="descending" id="cases">OMIC Cases</th>
                 <th class="govuk-table__header govuk-table__header--numeric" aria-sort="none" id="custody">Total Points while in Custody</th>
-                <th class="govuk-table__header govuk-table__header--numeric" aria-sort="none" id="licence">Anticipated Points while on Licence</th>
+                <th class="govuk-table__header govuk-table__header--numeric" aria-sort="none" id="license">Anticipated Points while on Licence</th>
```
Comment (iainjchambers-moj) [inline app/views/includes/omic-organisation-overview.njk:7]: Isn't this the wrong way around?

File: app/views/includes/omic-organisation-overview.njk [hunk 2]
```diff
@@ -13,7 +13,7 @@
               <td class="govuk-table__cell" headers="name">{% if childOrganisationLevelDisplayText !== 'Team' and childOrganisationLevelDisplayText !== 'Offender Manager' %} <a class="govuk-link" href="/{{ workloadType }}/{{ childOrganisationLevel }}/{{ member.linkId }}">{{ member.name }}</a> {% else %} {{ member.name }} {% endif %}</td>
               <td class="govuk-table__cell" headers="cases total">{{ member.totalCases }}</td>
               <td class="govuk-table__cell" headers="custody total">{{ member.custodyPoints }}</td>
-              <td class="govuk-table__cell" headers="licence total">{{ member.licencePoints }}</td>
+              <td class="govuk-table__cell" headers="license total">{{ member.licencePoints }}</td>
```
Comment (iainjchambers-moj) [inline app/views/includes/omic-organisation-overview.njk:16]: Same as above

File: app/views/includes/omic-organisation-overview.njk [hunk 3]
```diff
@@ -22,7 +22,7 @@
             <th class="govuk-table__cell" headers="name om">Total/ Average</th>
             <th class="govuk-table__cell" headers="cases total">{{ overviewDetails.totals.totalTotalCases }}</th>
             <th class="govuk-table__cell" headers="custody total">{{ overviewDetails.totals.totalCustodyPoints }}</th>
-            <th class="govuk-table__cell" headers="licence total">{{ overviewDetails.totals.totalLicencePoints }}</th>
+            <th class="govuk-table__cell" headers="licence total">{{ overviewDetails.totals.totalLicensePoints }}</th>
```
Comment (iainjchambers-moj) [inline app/views/includes/omic-organisation-overview.njk:25]: Same as above

File: app/views/includes/omic-organisation-overview.njk (hunk 1 above)
Comment (sambailey-moj) [reply app/views/includes/omic-organisation-overview.njk:7]: changed the id's by mistake in this file

File: app/views/includes/omic-organisation-overview.njk (hunk 3 above)
Comment (sambailey-moj) [reply app/views/includes/omic-organisation-overview.njk:25]: changed the id's by mistake in this file

**Review by iainjchambers-moj — APPROVED** [review]

**Pull request merged**